pages/common/gixy.md

```diff
@@ -0,0 +1,20 @@
+# gixy
+
+> Analyze nginx configuration.
```

Comment:
Perhaps this will read better?
```suggestion
> Analyze nginx configuration files.
```

Reply:
Singular is correct in this case.

Reply:
That depends. Many configurations will have multiple nested configuration files in directories such as `sites-available` and `conf.d`. In addition, if we used the singular here we'd need to reword it to be `Analyze an nginx configuration file` to add an article, though this would imply that it can only check a single configuration file, @Aracki

Reply:
Thanks, @sbrl for noticing this. I checked it, and it can check multiple files. So I modify the title and last example.